moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/current_state_monitor.h

```diff
@@ -168,10 +168,30 @@ class CurrentStateMonitor
    *  @return Returns the map from joint names to joint state values*/
   std::map<std::string, double> getCurrentStateValues() const;
 
-  /** @brief Wait for at most \e wait_time seconds (default 1s) for a robot state more recent than t
+  /** @brief Wait for at most \e wait_time seconds (default 1 s) for a robot state more recent than \e t
+   *  @return true on success, false if an up-to-date robot state message wasn't received within \e wait_time
+   */
+  inline bool waitForCurrentState(const ros::Time t = ros::Time::now(), double wait_time = 1.0) const
+  {
+    return waitForCurrentState(robot_model_->getJointModels(), t, wait_time);
+  }
+
+  /** @brief Wait for at most \e wait_time seconds (default 1 s) for a robot state more recent than \e t,
+   *         containing the joints in \e group
    *  @return true on success, false if up-to-date robot state wasn't received within \e wait_time
    */
-  bool waitForCurrentState(const ros::Time t = ros::Time::now(), double wait_time = 1.0) const;
+  inline bool waitForCurrentState(const std::string& group, const ros::Time t = ros::Time::now(),
```

Comment:
another convenience overload with the `JointModelGroup` would be  ... well  ... convenient